File: check-balance/PYTHON/balance.py
import requests
import urllib.request
import urllib.parse

username='enter your Username'
password='Enter your password'
request='balance'

req = sendSMS(
    username=username,
    password= password,
    request=request,
)

def sendSMS(username, password, request):
    params = {
        'username':username,
        'password': password,
        'request':request,
        
    }
    
    url = 'http://www.bongolive.co.tz/api/request.php?{params}'.format(params=urllib.parse.urlencode(params))
    
    return requests.get(url, params=params) 

print (req)

File: check-balance/PHP/balance.php
?php

define ("URL_API_DOMAIN", "http://www.bongolive.co.tz/api/");
$username = "Enter your username";
$password = "Enter your password";
$request = "balance";

$posturl = "http://www.bongolive.co.tz/api/request.php?username=$username&password=$password&request=$request";

$ch=curl_init();
curl_setopt($ch, CURLOPT_URL, $posturl);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
$response = curl_exec($ch);

echo "response $response";

?>